### ISSUE TITLE
fix size

### ISSUE THREAD

**Issue by kariminf** [assem-ch/snowball#3]
For Arabic,
The size in C is 1+2*(number of characters)
The size in Java is (number of characters)

**Comment by kariminf** [comment]
The problem is in C compiler; it uses a function named SIZE() to get the size of a string; but this function returns the number of bytes including the end of string (\0).
So, if we want to get the number of characters, we have to use "mbstowcs"
See: [rosettacode](http://rosettacode.org/wiki/String_length#Character_Length_9)

```
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main()
{
    setlocale(LC_CTYPE, "");
    char moose[] = "møøse";
    printf("bytes: %d\n", sizeof(moose) - 1);
    printf("chars: %d\n", (int)mbstowcs(0, moose, 0));

    return 0;
}
```

The result is 7, 5

**Comment by assem-ch** [comment]
fixed by PR #4
